pkg/operator/csi/credentialsrequestcontroller/credentials_request_controller.go

```diff
@@ -21,7 +22,11 @@ import (
 
 // CredentialsRequestController is a simple controller that maintains a CredentialsRequest static manifest.
 // It uses unstructured.Unstructured as currently there's no API type for this resource.
-// TODO: the functionality in this controller should be moved to the StatisResourceController.
+// This controller produces the following conditions:
+// <name>Available: indicates that the secret was successfully provisioned by cloud-credential-operator.
+// <name>Progressing: indicates that the secret is yet to be provisioned by cloud-credential-operator.
+// <name>Degraded: produced when the sync() method returns an error.
+// TODO: applying static credentials should be incorporated to StatisResourceController.
```

Comment:
This TODO is no longer valid

Reply:
@deads2k with this PR this controller does a little than just applying a static file. Do you still want to have the StaticResourceController to apply CredentialsRequest objects?